socorro/external/boto/connection_context.py

```diff
@@ -202,10 +197,48 @@ def load_file(self, path):
 
         """
         try:
-            resp = self.client.get_object(Bucket=self.config.bucket_name, Key=path)
+            resp = self.client.get_object(Bucket=bucket, Key=path)
             return resp["Body"].read()
         except self.client.exceptions.NoSuchKey:
             raise KeyNotFound(
-                "%s (bucket=%r key=%r) not found, no value returned"
-                % (id, self.config.bucket_name, path)
+                f"(bucket={bucket!r} key={path}) not found, no value returned"
+            )
+
+    def list_objects_paginator(self, bucket, prefix):
```

Comment:
I moved some code from other places to here so that it's not spread out across the code base. This is used by the webapp command that looks for crash reports we collected, but didn't process.